arch/arm/src/kinetis/kinetis_i2c.c

```diff
@@ -191,7 +191,7 @@ static struct kinetis_i2cdev_s g_i2c0_dev =
   .config     = &kinetis_i2c0_config,
   .refs       = 0,
   .lock       = NXMUTEX_INITIALIZER,
-  .wait       = NXSEM_INITIALIZER(0, PRIOINHERIT_FLAGS_DISABLE),
+  .wait       = SEM_INITIALIZER(0),
```

Comment:
I still think that we need to use `NXSEM_INITIALIZER(0, 0)` for kernel code, just from style point of view, but I will not block PR because of this.

Reply:
But, it isn't a issue to use a macro or function which doesn't start with NX or nx prefix, if the the function/macro doesn't touch errno or other per task data(e.g. file handle).